pulp_docker/app/registry.py

```diff
@@ -2,18 +2,30 @@
 import os
 
 from aiohttp import web, web_exceptions
+from aiohttp.client_exceptions import ClientResponseError
 from django.conf import settings
 from django.core.exceptions import ObjectDoesNotExist
+from django.db import IntegrityError, transaction
 from gettext import gettext as _
 from multidict import MultiDict
 
-from pulpcore.plugin.models import ContentArtifact
+from pulpcore.plugin.models import Artifact, ContentArtifact, Remote
```

Comment:
Super nitpick (only do if you go back in here anyway). Technically, pulpcore should be considered a 3rd party library and should be included in the second block after multidict. pulp_docker is the only internal import.